symfony/framework-bundle/5.2/public/index.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use App\Kernel;
+use Symfony\Component\HttpFoundation\Request;
+
+require_once ($_SERVER['APP_VENDOR_DIR'] = dirname(__DIR__).'/vendor').'/symfony/runtime/autoload.php';
```

Comment:
Sorry if I missed it - how is the APP_VENDOR_DIR used? I mean, why do we need to set it?

Reply:
`APP_VENDOR_DIR` is used to locate the autoload.php file.
It's usually not needed because the runtime can figure by doing `../../` from inside the vendor dir - but vendors are not always there. (e.g. when using the `link` util provided in the `symfony/symfony` repo.

Reply:
I removed using `APP_VENDOR_DIR` since that's useful only to ppl that use the `./link` script, which is far from the majority.